chemqulacs/vqe/vqeci.py

```diff
@@ -416,11 +421,20 @@ def kernel(self, h1, h2, norb, nelec, ecore=0, **kwargs):
             self.fermionic_hamiltonian,
         )
         # Set initial Quantum State
-        occ_indices = list(range(self.n_electron))
+
+        for m in range(self.n_electron, 9999):
```

Comment:
I thought about it a bit math.comb is biggest for comb(2n, n) so we only need to check until self.n_electron *2. Maybe we should also check to see if we have found a solution. In the current implementation, when there is no solution m would just be 9999. and continue the program. 
Another option would be to write math.comb(m,n) as a polynomial and look for the smallest real root and take the round-up result. With this, the implementation might be slightly faster, but since I suspect that self.n_electron is a small number, the time saved might be minimal if even notice. So maybe its not worth spending so much time on it.

Reply:
I added code to throw an exception if no solution was found.